misk/src/main/kotlin/misk/concurrent/ExecutorServiceModule.kt

```diff
@@ -0,0 +1,34 @@
+package misk.concurrent
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder
+import misk.inject.KAbstractModule
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+import kotlin.reflect.KClass
+
+/**
```

Comment:
remove empty comment, though would be nice to have docs for the methods

Reply:
I removed this this time. :)